src/main/java/Models/FileWrapper.java

```diff
@@ -0,0 +1,42 @@
+package Models;
+
+import Services.LogService;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileWrapper {
```

Comment:
What is the actual use-case for such a FileWrapper?

Reply:
We think it would be useful to store and manage logs per file, and this is where FileWrapper comes in. There is later a mapping of project files to project, where each project file has their own FileWrapper, which makes it pretty easy to find all logs for project AND to find all logs per file.
This mapping log --> file could have been done in another way (e.g. using a lot of hash maps for all relations: project to file, file to list of logs), but having a separate class seems to be a more elegant solution since it avoids repetition (for example, if file name changes you would need to remember to change it in both places).